requirements/pip.txt

```diff
@@ -95,3 +95,6 @@ django-cors-middleware==1.3.1
 
 # User agent parsing - used for analytics purposes
 user-agents<1.2.0
+
+# Required only in development and linting
+django-debug-toolbar==1.11
```

Comment:
I think we should put this in another file like https://github.com/stsewd/readthedocs.org/blob/b856e791b6be6cff68a98efd1422f30136c2d856/requirements/testing.txt#L27-L29? Mostly because this will be installed in production too, we are not going to activate it, but looks like a dead package to have.

Reply:
The reason I put it here is that it is required for testing, development, and linting. Currently only the default `requirements.txt` file (which then loads `pip.txt`) is used in development according to [our documentation](https://docs.readthedocs.io/en/stable/install.html#get-and-run-read-the-docs).